Specs/Renderer/ShaderProgramSpec.js

```diff
@@ -355,6 +358,27 @@ defineSuite([
         expect(renderFragment(context, sp)).toEqual([255, 255, 255, 255]);
     });
 
+    it('creates duplicate uniforms if precision of uniforms in vertex and fragment shader do not match', function() {
```

Comment:
In the test I'm using czm_viewport. Is there a reason to use czm_sunDirectionWC besides being the first issue?

Also the czm_sunDirectionWC bug won't actually happen anymore. It was added as a dependency in ShaderSource despite being in a comment. But now inline comments are removed.